ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicyCriteriaKeys.tsx

```diff
@@ -1,41 +1,80 @@
 import React from 'react';
-import { Title, Divider, Flex } from '@patternfly/react-core';
+import { groupBy } from 'lodash';
+import { Title, Divider, Flex, Text } from '@patternfly/react-core';
+
+import { policyCriteriaCategories } from 'messages/common';
+import type { PolicyCriteriaCategoryKey } from 'messages/common';
 
 import PolicyCriteriaCategory from './PolicyCriteriaCategory';
+import { Descriptor } from './policyCriteriaDescriptors';
+
+type CriteriaDomain =
+    | 'Image criteria'
+    | 'Workload configuration'
+    | 'Workload activity'
+    | 'Kubernetes resource operations';
+
+const criteriaDomains: Record<PolicyCriteriaCategoryKey, CriteriaDomain> = {
```

Comment:
What is pro and con of unit test (can be follow up) to assert that this object and `policyCriteriaCategories` are compatible, if code depends on that assumption?

Reply:
Yeah, that is a possibility. I wonder if it is possible to enforce that cleanly via TypeScript, sort of like a bi-directional record? Probably not critical for this change but would be interested to see what the options are.

Reply:
For your awareness, not necessarily a recommendation, there are unit tests for consistent capitalization:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.test.ts#L5